src/PendingBets.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { Heading, Button } from 'rebass';
+import styled from 'styled-components';
+import { store } from './store';
+import { Table } from './Table';
+import { Prime } from './types';
+
+const StakeButton = styled(Button)`
+  padding: 0 8px 0 12px;
```

Comment:
I suppose you wanted to do a reverse to align emoji to the center of the button, though it will affect button with text label
```
padding: 8px 10px 12px;
```